### PR TITLE
MC-12998: Added documentation for the Edit and align the list and cre…

### PR DESCRIPTION
**Pull request by fyrnaga**
…ate with the changes made

#### Changes made

- Made correction on list/get and create
- Added the edit documentation

#### Related PRs
- PR # https://github.com/cloudops/cloudmc-api-docs/pull/273
- PR # https://github.com/cloudops/cloudmc-stackpath-plugin/pull/23

### CODE REVIEW

File: source/includes/stackpath/_network_policy_rules.md [hunk 1]
```diff
@@ -52,17 +52,22 @@ curl -X GET \
```
Comment (y0geshdev) [inline source/includes/stackpath/_network_policy_rules.md]:
```suggestion
<code>GET /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/networkpolicyrules</code>
```

File: source/includes/stackpath/_network_policy_rules.md [hunk 2]
```diff
@@ -52,17 +52,22 @@ curl -X GET \
 
 Retrieve a list of all network policy rules in a given [environment](#administration-environments).
 
+
+Query Params | &nbsp;
+---- | -----------
+`workloadId`<br/>*string* | The workload ID to get the network policy rules for. It is optional.
+
```
Comment (patrickspensieri) [inline source/includes/stackpath/_network_policy_rules.md:59]: I missed the query params in my docs PR, thanks for catching that. We can also filter by `type`.

```suggestion
 `type`<br/>*string* | The type of network policy rule, either `INBOUND` or `OUTBOUND`. It is optional.
```

**Review by patrickspensieri — APPROVED** [review]

File: source/includes/stackpath/_network_policy_rules.md [hunk 3]
```diff
@@ -52,17 +52,22 @@ curl -X GET \
...
 Attributes | &nbsp;
 ------- | -----------
-`id`<br/>*string* | The ID of the network policy rule, in the form `workloadId/hashCode`.
+`id`<br/>*string* | The ID of the network policy rule, in the form `networkProfileId/type/hashCode/occurrence`.
```
Comment (y0geshdev) [inline source/includes/stackpath/_network_policy_rules.md:62]: Are we using `occurrence` as well, as part of network policy Id?

File: source/includes/stackpath/_network_policy_rules.md (hunk 1 above)
Comment (y0geshdev) [reply source/includes/stackpath/_network_policy_rules.md]: Adding this suggestion @patrickspensieri PR.

File: source/includes/stackpath/_network_policy_rules.md (hunk 3 above)
Comment (fyrnaga) [reply source/includes/stackpath/_network_policy_rules.md:62]: yep.. if there is 2 records with the same hash. This avoid error in the client. That is part of my last commit on my PR

**Review by razine-bensari — APPROVED** [review]

**Pull request merged**